packages/front-end/components/Billing/PaymentInfo.tsx

```diff
@@ -0,0 +1,264 @@
+import { useState, useEffect, useCallback } from "react";
+import { Flex, Text } from "@radix-ui/themes";
+import { CiCreditCard1 } from "react-icons/ci";
+import { PaymentMethod } from "shared/src/types/subscriptions";
+import { FaCreditCard } from "react-icons/fa";
+import { FaBuildingColumns } from "react-icons/fa6";
+import { useAuth } from "@/services/auth";
+import { useUser } from "@/services/UserContext";
+import track from "@/services/track";
+import { isCloud } from "@/services/env";
+import { GBAddCircle } from "../Icons";
+import Callout from "../Radix/Callout";
+import Badge from "../Radix/Badge";
+import MoreMenu from "../Dropdown/MoreMenu";
+import DeleteButton from "../DeleteButton/DeleteButton";
+import Modal from "../Modal";
+import LoadingSpinner from "../LoadingSpinner";
+import { StripeProvider } from "../Billing/StripeProvider";
+import Tooltip from "../Tooltip/Tooltip";
+import Button from "../Radix/Button";
+import AddPaymentMethodModal from "./AddPaymentMethodModal";
+
+export default function PaymentInfo() {
+  const [paymentMethodModal, setPaymentMethodModal] = useState(false);
+  const [defaultPaymentMethod, setDefaultPaymentMethod] = useState<
+    string | undefined
+  >(undefined);
+  const [error, setError] = useState<string | undefined>(undefined);
+  const [paymentMethods, setPaymentMethods] = useState<PaymentMethod[]>([]);
+  const [loading, setLoading] = useState(false);
+  const { subscription, organization } = useUser();
+  const { apiCall } = useAuth();
+  // TODO: Remove once all orgs have moved license info off of the org - only limit by isCloud()
+  // The licenseKey is required to look up payment methods
+  const canShowPaymentInfo = isCloud() && !!organization.licenseKey;
```

Comment:
Do we want to add a feature flag check here?  That way we can test with our prod stripe key first before opening it up to everyone.